### PR TITLE
Assignment-one, thanks markush

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -56,6 +58,7 @@ docs/_build/
 target/
 
 
+<<<<<<< HEAD
```
Comment (MarkusH) [inline .gitignore]: Some remaining merge conflicts.

File: .gitignore [hunk 2]
```diff
@@ -80,3 +83,5 @@ Temporary Items
 .apdisk
 
 
+=======
```
Comment (MarkusH) [inline .gitignore]: Done here as well.

**Comment by MarkusH** [comment]
Please don't commit the .pyc files (or better, the entire **pycache** folders) and the db.sqlite3 files.

File: guestbook/records/models.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here.
+class Record(models.Model):
+    class Meta:
```
Comment (MarkusH) [inline guestbook/records/models.py:7]: The recommendation in Django itself says to put `class Meta` after the fields: https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style but that's up for everybody to decide on their own.

File: guestbook/records/models.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        verbose_name_plural = 'records'
+
+    EMAIL_FORMAT = ('you@example.com')
+
+    name = models.CharField(max_length=200, default='Your awesome name') 
+    comment = models.CharField(max_length =2000)
```
Comment (MarkusH) [inline guestbook/records/models.py:13]: Do you really want a `CharField` or are you looking for a `TextField`?

File: guestbook/records/views.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+from django.http import Http404
+
+from .models import Record
+from .forms import RecordForm
+
+class RecordCreateView(CreateView):
+    model = Record
+    template_name = 'records.html'
+    form_class = RecordForm
+    success_url = '/'
```
Comment (MarkusH) [inline guestbook/records/views.py:12]: You might want to consider using a URL name here, e.g. `record-list`.

File: guestbook/records/models.py (hunk 3 above)
Comment (ghost) [inline guestbook/records/models.py:7]: :hand:

**Comment by ghost** [comment]
MakrusH you are reviewing my old commit,  the latest one is assignment one without test. I will try to pull request again. I took your note on charfield and class meta

File: guestbook/records/views.py (hunk 5 above)
Comment (ghost) [inline guestbook/records/views.py:12]: This my old commit. I am sure I fixed this one already

**Pull request closed without merging**